RevenueCatUI/Data/Strings.swift

```diff
@@ -98,6 +103,22 @@ extension Strings: CustomStringConvertible {
 
         case .setting_restored_customer_info:
             return "Setting restored customer info"
+
+        case .executing_external_purchase_logic:
+            return "Will execute custom StoreKit purchase logic provided by yourapp. " +
+            "No StoreKit purchasing logic will be performed by RevenueCat. " +
+            "You must use `.handlePurchase` on your `PaywallView`."
```

Comment:
Are we allowing `handlePurchase` and `handlePurchase` to be used in our SwiftUI view modifiers and in UIKit? If so, we maybe shouldn't referencing `PaywallView` since it wouldn't work in all contexts? 🤔

Reply:
Andy said to only worry about SwiftUI for V1, then decide what to do next...